client/components/common/TestPlanResultsTable/utils.js

```diff
@@ -1,15 +1,36 @@
 export const calculateAssertionsCount = testResult => {
-    const passedAssertionsCount = testResult.scenarioResults.reduce(
+    let passedAssertionsCount = testResult.scenarioResults.reduce(
         (acc, scenarioResult) =>
             acc + scenarioResult.assertionResults.filter(e => e.passed).length,
         0
     );
 
-    const failedAssertionsCount = testResult.scenarioResults.reduce(
+    let failedAssertionsCount = testResult.scenarioResults.reduce(
         (acc, scenarioResult) =>
             acc + scenarioResult.assertionResults.filter(e => !e.passed).length,
         0
     );
 
+    // For each command, a check for high negative and moderate negative impacts, so 2
+    const maxUnexpectedBehaviorsPassCount =
+        testResult.scenarioResults.length * 2;
+
+    // These are equivalent to failures
+    const unexpectedBehaviorCount = testResult.scenarioResults.reduce(
+        (acc, scenarioResult) =>
+            acc +
+            (scenarioResult.unexpectedBehaviors.some(
+                e => e.impact === 'HIGH' || e.impact === 'MODERATE'
+            )
+                ? 1
+                : 0),
+        0
+    );
+
+    const additionalPassedAssertions =
+        maxUnexpectedBehaviorsPassCount - unexpectedBehaviorCount;
+    passedAssertionsCount = passedAssertionsCount + additionalPassedAssertions;
+    failedAssertionsCount = failedAssertionsCount + unexpectedBehaviorCount;
```

Comment:
#738 does highlight unexpected behaviors as failures. 

> Instead, let any other negative behavior trigger failure for one of the above ... assertions

> if a high impact negative behavior were present for one command, then the MUST-have behavior support would drop from 100% (4/4) to 75% (3/4)

In that if a moderate or high severity behavior is found or not, it indicates a fail or pass respectively on the new static assertion, `Other behaviors that create <impact level> negative-impacts are not exhibited`. Does that make you view this differently?

Reply:
Although further separation of them isn't an issue and could prove useful, in supporting potential future requirements. Just in the likely case, different meanings become applied based on what the the severity level(s) are. So I can still workshop an alternate representation of this.

Reply:
Sorry I didn't quite follow that! 

Maybe I can try to restate my point above.

I feel like we should either rename our assertions or rename this weighted point system with the same name. I don't think we should use the same term for both things. They're quite different and it would get confusing. It doesn't have to be totally different, just different enough to tell them apart. Maybe we can call them "weightedAssertions" or something. Do you think that would be doable?

Reply:
> They're quite different and it would get confusing

You are right that these impact behaviors aren't assertions in the same way the system has been doing it. I've been seeing these as similar to if those lines were included in the source `assertions.csv` file but it's definitely not. So I understand the want for a clearer name here.

> Maybe we can call them "weightedAssertions" or something. Do you think that would be doable?

Does https://github.com/w3c/aria-at-app/pull/849/commits/849997d4c6b29adb4c1e890e3a691793473beba0 address this concern?